### ISSUE TITLE
type `image` is unused (unused)

### ISSUE THREAD

**Issue by ringods** [golangci/golangci-lint#1193]
As documented in the section on "False Positives", I report what I think is an occurrence of this. 

There is a type defined here:

https://github.com/terra-farm/terraform-provider-virtualbox/blob/e64dd8d729d93e2ea2df87460d7fbc35b7e451e3/virtualbox/image.go#L20-L29

And it is referred to in the same file later on:

https://github.com/terra-farm/terraform-provider-virtualbox/blob/e64dd8d729d93e2ea2df87460d7fbc35b7e451e3/virtualbox/image.go#L73

But when running this tool, I get this output reported:

```
virtualbox/image.go:20:6: type `image` is unused (unused)
```

I assume this is incorrect.

**Comment by sayboras** [comment]
@ringods the second one is receiver pointer, just want to check if you have other place, which actually create instance of _image_ struct ?

**Comment by iwankgb** [comment]
@ringods do you still consider it a bug or can we close the issue?

**Comment by ringods** [comment]
Well, I think the linter detected correctly that the type was never instantiated. So I'll close it.
